ambry-router/src/main/java/com.github.ambry.router/GetBlobOperation.java

```diff
@@ -393,6 +421,10 @@ void completeRead() {
     private void updateChunkingAndSizeMetricsOnSuccessfulGet() {
       routerMetrics.getBlobSizeBytes.update(bytesWritten);
       routerMetrics.getBlobChunkCount.update(numChunksTotal);
+      if (hasRange) {
+        routerMetrics.getBlobWithRangeSizeBytes.update(bytesWritten);
```

Comment:
I have removed the latency, total time, and range request chunk count metrics.  I kept the range request size and added a total blob size metric that will be populated on successful range requests.  I think these two metrics will be helpful for measuring use patterns.